app/assets/javascripts/modules/help.js.coffee

```diff
@@ -61,30 +61,61 @@ ns = @edsc.models.page
       showNext: true
       },{
       title: "Facets"
-      content: "<p>Refine your search further with available facets, such as:
-                <div style='margin-left: 15px;'><ul style='list-style-type: disc;'>
-                  <li>Features - subsettable, near-real-time or has imagery</li>
+      content: '<p>Refine your search further with available facets, such as:
+                <div class="tour-indent"><ul class="bullet-list">
```

Comment:
Can you also move this `ul` down to a new line? I don't really like having block level elements on the same line. So I think the markup should look like this:
```
<div class="tour-list">
  <ul class="bullet-list">
    <li>blah blah</li>
  </ul>
</div>
````
I think that makes things easier to read and see the structure.